bin/qserv-test-integration.py

```diff
@@ -67,7 +67,12 @@ def _parse_args():
         formatter_class=argparse.ArgumentDefaultsHelpFormatter
     )
 
+    
     parser = logger.add_logfile_opt(parser)
+    parser.add_argument('-q', action='store', dest='qserv_server', default='',
+                        help='qserv server')
```

Comment:
It's a bit confusing, is "qserv server" what we call now "master server" (mysql) or is it a czar that is used to run queries? Is it a host name or a name of the node in CSS?